db/schema.rb

```diff
@@ -19,6 +19,25 @@
     t.text 'text'
     t.datetime 'created_at', null: false
     t.datetime 'updated_at', null: false
+    t.bigint 'user_id'
```

Comment:
- [ ] Kindly ensure to use _integer_ attribute as specified in the ERD, instead of _bigint_.